### DISCUSSION TITLE
Hello I m new here Since yesterday I [Oct 22]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4238] [Discord Threads]
> 🙋‍♂️ **DtNeo**  
#### Thanks for helps  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1030824571324076113/unknown.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **DtNeo**  
#### Ok I saw "/" make it wrong.  
#  
> 🙋‍♂️ **DtNeo**  
#### it's solved for me. have a good week end  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1030824571584139315)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
